api/dist/kernel.json

```diff
@@ -28,6 +28,10 @@
       "Condition": "Development",
       "Value": { "Fn::If": [ "Autoscale", "true", "false" ] }
     },
+    "AwsAccount": {
```

Comment:
Ah I believe this is a remnant of something I was doing where I needed the account ID in Rack. Can be safely removed. Make sure it's out of the docker-compose.yml too